source/deployment/sso-saml-onelogin.rst

```diff
@@ -0,0 +1,127 @@
+    - **Recipient**: ``https://<your-mattermost-url>/login/sso/saml``
+    - **ACS (Consumer) URL**: ``https://<your-mattermost-url>/login/sso/saml``
+    - **ACS (Consumer) URL Validator**: ``https:\/\/<your-mattermost-url>\/login\/sso\/saml``
+
+  .. image:: ../../source/images/onelogin_3_configuration_1.png
+
+  b. Paste the Public Key that you generated earlier into the **SAML Encryption** field.
+
+  .. image:: ../../source/images/onelogin_4_configuration_2.png
+
+  c. Click **Save**.
+
+3. Enter the attribute parameters.
+
+  Attribute parameters map attributes between OneLogin and Mattermost. For more information on which attributes are configurable, see our :ref:`documentation on SAML configuration settings <saml-enterprise>`.
+
+  *Email* and *Username* attributes are required. For Mattermost servers running version 3.3 and earlier, *FirstName* and *LastName* attributes are also required.
+
+  a. Click the **Parameters** tab.
+  b. Click **Add Parameter**.
+
+  .. image:: ../../source/images/onelogin_5_parameters_add.png
+
+  c. In the **Field name** field, enter an attribute parameter such as ``Email``.
+  d. Select the **Include in SAML assertion** checkbox.
+  e. Click **Save**
+
+  .. image:: ../../source/images/onelogin_6_parameters_add_2.png
+
+  f. Click **Edit**.
+  g. In the **Value** field, select the OneLogin value that corresponds to the attribute parameter.
+
+  .. image:: ../../source/images/onelogin_7_parameters_add_3.png
+
+  h. Repeat steps b through g to add the *Username* attribute and any other attributes that you need.
+
+    After you've added all the attributes you want to use, the parameter list should look similar to the following image:
+
+    .. image:: ../../source/images/onelogin_8_parameters_add_4.png
+
+4. Copy the SSO information.
+  a. Click the **SSO** tab.
+  b. Copy the values in the **Issuer URL** and **SAML 2.0 Endpoint (HTTP)** fields and save them for use later.
+
+  .. image:: ../../source/images/onelogin_9_sso.png
+
+  c. Click **View Details** to view the X.509 certificate.
+  d. Make sure that the **X.509 PEM** option is selected in the drop-down.
+
+  .. image:: ../../source/images/onelogin_10_sso_certificate.png
+
+  e. Click **DOWNLOAD** and save the file in a convenient location for use later.
+5. Save all your changes.
+
+Configure SAML for Mattermost
+-----------------------------
+
+1. Sign into Mattermost as a System Administrator.
+2. Go to **System Console > Authentication > SAML**.
+  a. In the **SAML SSO URL** field, paste the value for the OneLogin *SAML 2.0 Endpoint (HTTP)* that you copied earlier.
+  b. In the **Identity Provider Issuer URL** field, paste the value for the OneLogin *Issuer URL* that you copied earlier.
+  c. In the **Identity Provider Public Certificate** field, upload the OneLogin X.509 PEM certificate file that you downloaded earlier.
+
+  .. image:: ../../source/images/okta_10_mattermost_basics.png
```

Comment:
I think it's because the images already exist in the repo and are not included in this PR, so they don't show up.